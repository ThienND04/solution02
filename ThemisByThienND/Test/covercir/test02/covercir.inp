5
417 138  264
560 192  533
558 525  129
309 42  143
917 448  69
