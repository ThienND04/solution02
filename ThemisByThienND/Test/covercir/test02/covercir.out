235
300
317
172
484
